src/Page/CKANRegistryPage.php

```diff
@@ -29,8 +36,18 @@ class CKANRegistryPage extends Page
 
     public function getCMSFields()
     {
-        $this->beforeUpdateCMSFields(function (FieldList $fields) {
-            $fields->addFieldToTab('Root.Data', ResourceLocatorField::create('Thing'));
+        $resource = $this->DataResource();
```

Comment:
`$this` is bound to closures in PHP 5.4 onwards, you don't need to pass it it into the closure like this